gpu_direct_rdma_access.c

```diff
@@ -838,6 +855,30 @@ struct rdma_device *rdma_open_device_server(struct sockaddr *addr)
     return NULL;
 }
 
+int rdma_reset_device(struct rdma_device *device)
+{
+    if (device->srq != NULL || !device->qpex || !device->mqpex) {
+        fprintf(stderr, "Method \"rdma_reset_device()\" could be executed only by server side!\n");
+        return 1;
```

Comment:
I think we should start returning proper errno values: https://man7.org/linux/man-pages/man3/errno.3.html

In this case I suggest to return with **EOPNOTSUPP**;